nodeApps/createPrivateRelease.sh

```diff
@@ -4,13 +4,28 @@
 
 set -e
 
+ENVIRONMENT=$1
+NO_METRIC=$2
+if [ $DEBUG != "" ]; then
+  SAFE_DEBUG_ECHO=echo
+fi
+
+# allow the shortbreaks' version $GHUSER:$GHPASS so ci keeps running
+if [ -z $GITHUB_USER ]; then
```

Comment:
I would like to see us using common Variables across the group for things like this. Perhaps something for the culture repo? For now, this is good for backwards compatibility.